stable/app/templates/cron.yaml

```diff
@@ -1,12 +1,15 @@
 {{- if .Values.cron.enabled }}
+{{- $fullName := include "app.fullname" . -}}
```

Comment:
These changes to `app` chart are not being used in the current PR, shall we take it up as a separate PR ?

Reply:
@rohilsurana These are absolutely necessary. 😓  
The cron template had issues, and it took a good amount of debugging to understand what is causing it and then modifying it for the same. During the course of which, the app-chart `0.4.0` is updated with the latest fixed cron changes as part of GitHub pr actions.
So this is necessary to go in here along with guardian changes.

Reply:
I see. Cool then.